src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java

```diff
@@ -64,6 +66,32 @@ public class RepositoryOptions extends OptionsBase {
           """)
   public PathFragment repoContentsCache;
 
+  @Option(
+      name = "repo_contents_cache_gc_max_age",
+      defaultValue = "14d",
+      documentationCategory = OptionDocumentationCategory.BAZEL_CLIENT_OPTIONS,
+      effectTags = {OptionEffectTag.BAZEL_INTERNAL_CONFIGURATION},
+      converter = DurationConverter.class,
+      help =
+          """
+          Specifies the amount of time an entry in the repo contents cache can stay unused before \
+          it's garbage collected. If set to zero, garbage collection is disabled.
+          """)
+  public Duration repoContentsCacheGcMaxAge;
+
+  @Option(
+      name = "repo_contents_cache_gc_idle_delay",
+      defaultValue = "5m",
+      documentationCategory = OptionDocumentationCategory.BAZEL_CLIENT_OPTIONS,
+      effectTags = {OptionEffectTag.BAZEL_INTERNAL_CONFIGURATION},
+      converter = DurationConverter.class,
+      help =
+          """
+          Specifies the amount of time the server must remain idle before garbage collection happens
+          to the repo contents cache. If set to zero, garbage collection is disabled.
```

Comment:
For the similarly named options, `0` actually means run immediately (while still respecting the other flag wrt enabled/disabled). Consider doing the same here to reduce confusion?

Reply:
addressed during import